src/amount.h

```diff
@@ -22,7 +22,7 @@ static const CAmount COIN = 100000000;
  * critical; in unusual circumstances like a(nother) overflow bug that allowed
  * for the creation of coins out of thin air modification could lead to a fork.
  * */
```

Comment:
what happened to
`static const CAmount CENT = 1000000;`
I believe this was used across different files?